h2o-r/h2o-package/R/frame.R

```diff
@@ -3794,6 +3794,30 @@ h2o.tolower <- function(x) .newExpr("tolower", x)
 h2o.toupper <- function(x) .newExpr("toupper", x)
 
 #'
+#' Searches for matches to argument ‘pattern’ within each element
+#  of a string column.
+#'
+#' This function has similar semantics as R's native grep function
+#' and it supports a subset of its parameters. Default behavior is
```

Comment:
Now that I think about it, its not. The function parameters should be enough for one to figure out what we have to offer.